Examples/Tests/electrostatic_sphere/analysis_electrostatic_sphere.py

```diff
@@ -142,6 +142,7 @@ def calculate_error(E_axis, xmin, dx, nx):
 print("L2 error along x-axis = %s" %L2_error_x)
 print("L2 error along y-axis = %s" %L2_error_y)
 print("L2 error along z-axis = %s" %L2_error_z)
+print("L2 error tolerance = %s" %l2_tolerance)
```

Comment:
These print statements were added to provide more output in cases when the checks fail. This was helpful in getting the new CI test working.